build.sbt

```diff
@@ -506,10 +505,9 @@ lazy val sbtProj = (project in file("sbt"))
     buildInfoKeys in Test := Seq[BuildInfoKey](
       // WORKAROUND https://github.com/sbt/sbt-buildinfo/issues/117
       BuildInfoKey.map((fullClasspath in Compile).taskValue) { case (ident, cp) => ident -> cp.files },
+      classDirectory in Compile,
+      classDirectory in Test,
     ),
-    connectInput in run in Test := true,
-    outputStrategy in run in Test := Some(StdoutOutput),
-    fork in Test := true,
```

Comment:
losing these breaks `sbtOn`, and the remote debugging was also useful when using `sbtOn`.

Reply:
I made `fork` a function, so I am guessing I can write sbtOn based on that?

Reply:
sure.

just make sure: it works, it sends its output immediately to the shell (e.g the prompt text), and it setups the remote debugging port.

Reply:
Fixed sbtOn.

Reply:
Thank you!